src/libraries/DrawAccumulatorLib.sol

```diff
@@ -59,11 +60,12 @@ library DrawAccumulatorLib {
 
             uint256 remainingAmount = integrateInf(_alpha, relativeDraw, newestObservation_.available);
             uint256 disbursedAmount = integrate(_alpha, 0, relativeDraw, newestObservation_.available);
+            uint256 remainder = newestObservation_.available - (remainingAmount + disbursedAmount);
 
             accumulator.drawRingBuffer[ringBufferInfo.nextIndex] = _drawId;
             accumulator.observations[_drawId] = Observation({
                 available: uint96(_amount + remainingAmount),
-                disbursed: uint168(newestObservation_.disbursed + disbursedAmount)
+                disbursed: uint168(newestObservation_.disbursed + disbursedAmount + remainder)
```

Comment:
We have `uint256 remainder = newestObservation_.available - (remainingAmount + disbursedAmount);`

`newestObservation_.disbursed + disbursedAmount + remainder` is equivalent to `newestObservation_.disbursed + disbursedAmount + newestObservation_.available - (remainingAmount + disbursedAmount)`

So `disbursedAmount` cancel each other, so we can simplify the equation with:
```suggestion
                disbursed: uint168(newestObservation_.disbursed + newestObservation_.available - remainingAmount)
```